### ISSUE TITLE
Workflow automation

### ISSUE THREAD

**Issue by CodeCrafter912** [linuxmuster/linuxmuster-linbo#170]
Hi @HappyBasher should I set up the workflow automation for this repo?

**Comment by HappyBasher** [comment]
Hi @CodeCrafter912, means that packages will be automatically published when a tag with v in front is released?

**Comment by CodeCrafter912** [comment]
Nope, the Packages will be compiled and uploaded to GitHub on every v tag.
You can them download and test them locally.
Once you're ready, you can manually trigger the upload to the lmn package server.

**Comment by HappyBasher** [comment]
Ok, do it. :-)

**Comment by CodeCrafter912** [comment]
I created a pull request regarding this: #173
